free-1.md

```diff
@@ -5,47 +5,69 @@
 This section describes who the project would serve and why a language might be a
 good way to meet their needs.
 
+A DSL may he helpful to write smart contracts (the core of applied blockchain technology). Currenlty the programming languages that can interact with the blockchain network are relativly new and not fully built out yet. When learning how to write in these languages, the syntax of writing a smart contract can be confusing at first. I hope to solve this problem by implmenting a DSL that makes writing simple smart contracts an easy task.
```

Comment:
I do not know how blockchain or smart contracts work, and I found myself lost trying to understand your project. I know the questions do not require you to explain the workings of your domain, but it would have been great if you had done so given how complex and esoteric the domain is.